content/en/user-manual/publishing/mobile/cordova.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Apache Cordova
+template: usermanual-page.tmpl.html
+position: 1
+---
+
+[Apache Cordova][1] is an open-source mobile development framework. It allows you to use standard web technologies - HTML5, CSS3, and JavaScript for cross-platform development. Applications execute within wrappers targeted to each platform, and rely on standards-compliant API bindings to access each device's capabilities such as sensors, data, network status, etc.
+
+You can use Cordova to natively wrap your PlayCanvas app. You can then publish it to the iOS App Store and Android's Google Play. Cordova can also generate executables compatible with macOS and Windows.
+
+## Installing Cordova
+
+To get started, follow the [instructions][2] for installing Cordova on your computer.
+
+## Creating a Project
+
+To create a project, issue the following command:
+
+```
+cordova create hello com.example.hello HelloWorld
+```
+
+`hello` is the folder in which the project is created. `com.example.hello` is the reverse domain-style identifier for your app. `HelloWorld` is the human readable title of your app (it is the name of the generated app icon, for example).
+
+So, for a game like [Master Archer][3], an appropriate command would be:
+
+```
+cordova create masterarcher com.playcanvas.masterarcher "Master Archer"
+```
+
+Once your project is created, you will find a file called `config.xml` in the project's root folder. Here you can configure or edit certain characteristics of your app. For example, you can optionally [set up custom icons][4] for your app, either globally or per-platform.
+
+## Adding your PlayCanvas App
+
+When you create a new Cordova project, it generates a skeleton web app in a folder called `www`. You can go ahead and delete everything in the `www` folder. Next, copy your PlayCanvas app files to this location.
+
+If you're building on the Engine without the Editor, copy your app files into `www` such that your `index.html` file is in the root.
+
+If you have built your app in the PlayCanvas Editor, hit the Publish button on the toolbar:
+
+![Publish Toolbar Button][5]
+
+Then select Download:
+
+![Download Zip][6]
+
+Then select the scenes to include and hit DOWNLOAD:
+
+![Download New Build][7]
+
+When you have downloaded the Zip file, extract its contents to the root of the `www` folder.
+
+Note that you can use the [PlayCanvas REST API][8] if you would like to automate this part of the process.
```

Comment:
Wrap the note in a box? 

![image](https://user-images.githubusercontent.com/16639049/98535464-10528800-227e-11eb-9f10-f966a3c4d2a2.png)